lib/src/functions/map.dart

```diff
@@ -24,10 +24,19 @@ final global = UnmodifiableListView([
 final module = BuiltInModule("map",
     functions: [_get, _merge, _remove, _keys, _values, _hasKey]);
 
-final _get = _function("get", r"$map, $key", (arguments) {
+final _get = _function("get", r"$map, $key, $keys...", (arguments) {
   var map = arguments[0].assertMap("map");
-  var key = arguments[1];
-  return map.contents[key] ?? sassNull;
+  var allKeys = [arguments[1], ...arguments[2].asList];
+  var intermediateKeys = allKeys.sublist(0, allKeys.length - 1);
+  for (var key in intermediateKeys) {
```

Comment:
Same as the other CL, you can use `.take()` to avoid an intermediate list.

Reply:
@jathak I think this comment still applies (replacing `sublist` with `take`).

Reply:
Whoops. Good catch